timestream_test.go

```diff
@@ -172,15 +173,36 @@ var (
 	}
 )
 
-type TimeStreamWriterMock struct {
-	timestreamwriteiface.TimestreamWriteAPI
+type TimeStreamWriterMock struct{}
+
+type TimeStreamQueryMock struct{}
+
+type PaginatorMock struct {
+	callCount int
 }
 
-type TimeStreamQueryMock struct {
-	timestreamqueryiface.TimestreamQueryAPI
+func NewPaginatorMock(client TimestreamQueryApi, params *timestreamquery.QueryInput, optFns ...func(*timestreamquery.QueryPaginatorOptions)) PaginatorApi {
+	return &PaginatorMock{}
 }
 
-func (t TimeStreamWriterMock) WriteRecords(input *timestreamwrite.WriteRecordsInput) (*timestreamwrite.WriteRecordsOutput, error) {
+func (p *PaginatorMock) HasMorePages() bool {
+	p.callCount++
+	// Return true for the first two calls, then false.
+	return p.callCount <= 2
+}
+
+func (p *PaginatorMock) NextPage(ctx context.Context, optFns ...func(*timestreamquery.Options)) (*timestreamquery.QueryOutput, error) {
```

Comment:
`HasMorePages()` and `NextPage()` are hacky but I just wanted to get the tests passing